geoips/plugins/modules/algorithms/visir/HNear_Constant_Contrast.py

```diff
@@ -0,0 +1,108 @@
+# # # This source code is subject to the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+"""Data algorithm for High Near constant contrast imagery product.
+
+Algorithm is based upon work by Zinke (2017) for improved NCC VIIRS DNB imagery.
+"""
```

Comment:
This description is insufficient. What is "High NCC" and how does it differ from what is in normal NCC?